recipes/protlearn/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "protlearn" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a9d7e6b1d0936614301e7a5e80ce6a3c2f71c5fff663c58838d051cc2fb250c7
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - pandas
+    - scikit-learn
+    - xgboost
+    - mlxtend
+    - biopython
+
+test:
+  imports:
+    - protlearn
+    - protlearn.dimreduction
+  requires:
+    - pytest
```

Comment:
If you aren't running any tests with `pytest` then you don't need it under `requires`, so just delete the `requires` section.

Reply:
Sounds good - pytest has been deleted from requires.